.spellignore

```diff
@@ -565,6 +565,10 @@ USD
 EUR
 CCO
 untagged
+<<<<<<< HEAD
```

Comment:
re Lines 568, 571, 589 - Should this be in here? Looks like merge conflict markers